src/main/kotlin/com/cognifide/gradle/aem/pkg/tasks/Deploy.kt

```diff
@@ -129,6 +136,17 @@ open class Deploy : Sync() {
         aem.notifier.notify("Package deployed", "${packages.fileNames} on ${instances.names}")
     }
 
+    private fun tailTaskNotification() {
+        val tail = aem.project.tasks.find { it is Tail }
+        if (tail is Tail && !tail.isRunning()) {
+            aem.logger.lifecycle("*****************************************************************************************")
+            aem.logger.lifecycle("*                                                                                       *")
+            aem.logger.lifecycle("* Notice: aemTail task is not running, consider starting it to easily monitor AEM logs. *")
```

Comment:
Avoid using task name / code details in UI, I mean 'aemTask'. Messages should be natural for humans 😉

Reply:
I mean, GAP notification `aem.notifier` ;  as fallback it is logging to console for CI usages